src/carousel/carousel.scss

```diff
@@ -64,6 +63,9 @@
   &-dots {
     @include iui-reset;
     @include iui-focus;
+    display: flex;
+    align-items: center;
+    flex: 2;
```

Comment:
Would it make sense to add some max-width for too wide carousels..?

Reply:
Added `max-width: 256px`

Reply:
good, but why 256? 1 dot is 28px and dots have no gap in between... Doesnt seem to divide nicely..?

Reply:
i just went with our size variables. Think I should use `28 * 9` or something?

Reply:
yeah idk.. our buttons sizes are really weird :/